tests/textcode/test_markup.py

```diff
@@ -48,190 +48,189 @@ def test_jsp_demarkup(self):
         test_file = self.get_test_loc(u'markup/java.jsp')
         result = list(markup.demarkup(test_file))
         expected = [
-            u' version="1.0" encoding="ISO-8859-1"?>\n',
```

Comment:
I have also think like that but I thought I have not allowed to touch expectations